modules/drone_odometry_local.py

```diff
@@ -66,36 +64,43 @@ def create_new(cls,
         return True, DroneOrientationLocal(cls.__create_key, orientation)
 
     @classmethod
-    def create_wrap(cls,
-                    orientation: drone_odometry.DroneOrientation):
+    def create_wrap(
+        cls, orientation: drone_odometry.DroneOrientation
+    ) -> "tuple[bool, DroneOrientationLocal | None]":
         """
         Wrap existing orientation.
         """
         return True, DroneOrientationLocal(cls.__create_key, orientation)
 
-    def __init__(self, class_private_create_key, orientation: drone_odometry.DroneOrientation):
+    def __init__(
+        self, class_private_create_key: object, orientation: drone_odometry.DroneOrientation
+    ) -> None:
         """
         Private constructor, use create() method.
         """
         assert class_private_create_key is DroneOrientationLocal.__create_key, "Use create() method"
 
         self.orientation = orientation
 
-# pylint: enable=too-few-public-methods
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        # TODO: Update common
+        return f"DroneOrientationLocal (YPR rad): {self.orientation.yaw}, {self.orientation.pitch}, {self.orientation.roll}"
```

Comment:
Keep __str__() function consistent with other modules. Either just return the string or always assign representation = f"...." and return representation